README.md

```diff
@@ -9,7 +9,8 @@
   <img src="docs/youki.png" width="230" height="230">
 </p>
 
-youki is an implementation of the [OCI runtime-spec](https://github.com/opencontainers/runtime-spec) in Rust, similar to [runc](https://github.com/opencontainers/runc).
+youki is an implementation of the [OCI runtime-spec](https://github.com/opencontainers/runtime-spec) in Rust, similar to [runc](https://github.com/opencontainers/runc).  
+Your feedback is welcome here. [#10](https://github.com/containers/youki/issues/10)
```

Comment:
How about this?
```suggestion
Your ideas are welcome [here](https://github.com/containers/youki/issues/10).
```

I worry that otherwise people will also report bugs in the general ideas issue instead of opening  a dedicated issue.

Reply:
I love your suggestion.